digits/model/tasks/caffe_train.py

```diff
@@ -1388,11 +1388,12 @@ def get_transformer(self):
             # XXX see issue #59
             channel_swap = (2,1,0)
 
-        if self.dataset.get_mean_file():
+        if self.get_mean_path() or self.mean_path is not None:
+
             if self.use_mean == 'pixel':
-                mean_pixel = self.get_mean_pixel(self.dataset.path(self.dataset.get_mean_file()))
+                mean_pixel = self.get_mean_pixel(self.dataset.path(self.get_mean_path()))
```

Comment:
if we're going to use `get_mean_path()` in model jobs I think it makes more sense to do `self.dataset.path(...)` from `get_mean_path()` otherwise we might be calling `self.dataset.path()` on non-dataset paths (which happens to do what we want in the case of an absolute path but we'd better not rely on it).